### PR TITLE
Added ollama gui interface

### PR DESCRIPTION
**Pull request by TwanLuttik**
This was already added but since the new release i think it didn't came through the merge i suppose.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -220,3 +220,4 @@ curl -X POST http://localhost:11434/api/generate -d '{
 - [Typescript UI](https://github.com/ollama-interface/Ollama-Gui?tab=readme-ov-file)
```
Comment (mxyng) [inline README.md:220]: The project is already linked. Perhaps you want to change the name?

**Review by mxyng — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (TwanLuttik) [reply README.md:220]: I see the name has been changed, but yes it would be what it was last time `Ollama GUI Interface`

Comment (mxyng) [reply README.md:220]: @mchiang0610 made the change so I'll let him weigh in

Comment (mchiang0610) [reply README.md:220]: Hey thank you. We just don’t want to confuse users that this is an official project. Thank you!

**Comment by mchiang0610** [comment]
Just want to say, I’m super appreciative of your work and think it’s amazing! Part of Ollama is its community and we don’t want to start playing favorites and prevent others from making their own too!! 

Thank you

**Pull request closed without merging**